### PR TITLE
[DOCS] Added Documentation of AudioStreamPlayer2D

### PR DESCRIPTION
**Pull request by Grosskopf**
That was comparatively copy&paste from AudioStreamPlayer3D :)

### CODE REVIEW

File: doc/classes/AudioStreamPlayer2D.xml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamPlayer2D" inherits="Node2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Plays audio in 2D.
 	</brief_description>
 	<description>
+		Plays audio that dampens with distance of the object to screen center.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer2D.xml]: Ref: style guide "avoid milk of the cow"

Should be "dampens with the object's distance *from* screen center"

File: doc/classes/AudioStreamPlayer2D.xml [hunk 2]
```diff
@@ -139,30 +144,40 @@
...
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in wich this sound plays.
 		</member>
 		<member name="attenuation" type="float" setter="set_attenuation" getter="get_attenuation">
+			Dampens Audio over distance with this as an exponent.
 		</member>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer2D.xml]: Add default value

File: doc/classes/AudioStreamPlayer2D.xml [hunk 3]
```diff
@@ -139,30 +144,40 @@
...
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in wich this sound plays.
 		</member>
 		<member name="attenuation" type="float" setter="set_attenuation" getter="get_attenuation">
+			Dampens Audio over distance with this as an exponent.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer2D.xml]: Audio shouldn't be capitalized.

File: doc/classes/AudioStreamPlayer2D.xml [hunk 4]
```diff
@@ -139,30 +144,40 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in wich this sound plays.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer2D.xml]: missing 'h'

File: doc/classes/AudioStreamPlayer2D.xml [hunk 5]
```diff
@@ -139,30 +144,40 @@
...
+			Bus on which this audio is playing.
 		</member>
 		<member name="max_distance" type="float" setter="set_max_distance" getter="get_max_distance">
+			Sets the absolute maximum of the soundlevel, in dB.
 		</member>
 		<member name="playing" type="bool" setter="_set_playing" getter="is_playing">
+			If [code]'true'[/code], audio is playing.
 		</member>
 		<member name="stream" type="AudioStream" setter="set_stream" getter="get_stream">
+			Decides what is playing.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer2D.xml]: This is very vague.  "The [AudioStream] object to be played."

File: doc/classes/AudioStreamPlayer2D.xml [hunk 6]
```diff
@@ -139,30 +144,40 @@
...
 		</member>
 		<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db">
+			Base volume without dampening.
 		</member>
 	</members>
 	<signals>
 		<signal name="finished">
 			<description>
+				Fires when the audio stopped playing.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer2D.xml]: "Emitted" is the preferred term, not "fires"

Also, you have mixed tense, should be "when the audio *stops* playing"

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/AudioStreamPlayer2D.xml (hunk 2 above)
Comment (Grosskopf) [reply doc/classes/AudioStreamPlayer2D.xml]: should i add default values everywhere? :)

File: doc/classes/AudioStreamPlayer2D.xml (hunk 4 above)
Comment (Grosskopf) [reply doc/classes/AudioStreamPlayer2D.xml]: ooh i burned the wich again :/

File: doc/classes/AudioStreamPlayer2D.xml (hunk 2 above)
Comment (cbscribe) [reply doc/classes/AudioStreamPlayer2D.xml]: Not needed everywhere, but there are a lot of properties where it's helpful to know what the default behavior is, so you know whether you need to change it or not.  For example, if autoplay were on by default, that would be useful to know.

File: doc/classes/AudioStreamPlayer2D.xml [hunk 7]
```diff
@@ -139,30 +144,40 @@
...
+			The [AudioStream] object to be played.
 		</member>
 		<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db">
+			Base volume without dampening.
 		</member>
 	</members>
 	<signals>
 		<signal name="finished">
 			<description>
+				Fires when the audio stops playing.
```
Comment (cbscribe) [inline doc/classes/AudioStreamPlayer2D.xml]: This still needs to be changed to "Emitted"

File: doc/classes/AudioStreamPlayer2D.xml [hunk 8]
```diff
@@ -77,6 +81,7 @@
 			<argument index="0" name="to_pos" type="float">
 			</argument>
 			<description>
+				Sets the position from wich audio will be played (in seconds).
```
Comment (Noshyaar) [inline doc/classes/AudioStreamPlayer2D.xml]: `which`

File: doc/classes/AudioStreamPlayer2D.xml [hunk 9]
```diff
@@ -139,30 +144,40 @@
...
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in which this sound plays.
 		</member>
 		<member name="attenuation" type="float" setter="set_attenuation" getter="get_attenuation">
+			Dampens audio over distance with this as an exponent.
 		</member>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game. Default value: [code]'false'[/code].
```
Comment (Noshyaar) [inline doc/classes/AudioStreamPlayer2D.xml]: Please remove quotes in [code] tags.

File: doc/classes/AudioStreamPlayer2D.xml [hunk 10]
```diff
@@ -139,30 +144,40 @@
...
 		</member>
 		<member name="bus" type="String" setter="set_bus" getter="get_bus">
+			Bus on which this audio is playing.
 		</member>
 		<member name="max_distance" type="float" setter="set_max_distance" getter="get_max_distance">
+			Sets the absolute maximum of the soundlevel, in dB.
```
Comment (Noshyaar) [inline doc/classes/AudioStreamPlayer2D.xml]: I'm not sure if the description is accurate.

Comment (Grosskopf) [reply doc/classes/AudioStreamPlayer2D.xml]: oh, weird, whatever i put there doesn't make much sense

**Pull request merged**